src/actions/ActionInputParameters.tsx

```diff
@@ -9,7 +9,7 @@ interface InputParametersProps {
 }
 
 function InputParameters({ data }: InputParametersProps) {
-  if (!data?.length) {
+  if (!data) {
```

Comment:
more strict alternative:
```javascript
if (!data || !Object.keys(data).length) {
```
but IMO not needed

Reply:
It's fine as it is 👍